.drone.yml

```diff
@@ -95,13 +95,32 @@ pipeline:
       event: deployment
       environment: [dev, demo]
 
-  deploy-to-qa:
+  deploy-to-qa-cs:
+    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+    environment:
+      - KUBE_NAMESPACE=cs-qa
+      - ENVIRONMENT=qa
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_management_ui_cs_qa
+    commands:
+      - source version.txt
+      - echo $VERSION
+      - cd kube-hocs-management-ui
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: qa
+      kube_namespace: cs-qa
```

Comment:
I don't think `kube_namespace` does anything here.

Reply:
This is a result of your comments in https://github.com/UKHomeOffice/hocs-casework/pull/323 and as per our private chat afterwards whether to use namespace or kube_namespace.
The context is identical to that of all the other projects (like casework above) for qa and prod.

Reply:
The `when:` part of a .drone.yml file merely governs whether a step will fire or be skipped; it doesn't govern what actually happens within the step. Drone itself has no concept of a kube namespace, so this entry in the .drone.yml file won't do anything and should be removed. The important part is the `KUBE_NAMESPACE=cs-qa` environment variable further up (which is consumed by the `deploy.sh` script in the corresponding kube- Git repo).

Reply:
After a discussion on the issue, it was decided to rename the environments to namespaces, for qa and prod. This will be processed in the next PR.
Approving this PR so that DEV can build correctly.